functions/src/Android Version/index.ts

```diff
@@ -46,6 +46,6 @@ function getSupportedAndroidVersions(res:any){
         const supportedVersion = {supportedVersion: verEndDateList}
         res.send(supportedVersion);
     }).catch(error => {
-        res.status(500).send("error getting supprted Android Versions: " + error);
+        res.status(400).send("error getting supported Android Versions: " + error);
```

Comment:
The errors in catch() statements can be 500. At the beginning of the function though we should have some data validation, like checking the format of CVE, format of Android version, etc. When validation fails, we return 400.